README.md

```diff
@@ -9,7 +9,7 @@ This is now alpha status.
 ## Install
 
 ```bash
-pip install git+git://github.com/common-workflow-language/python-cwlgen.git
+pip install git+git://github.com/EiffL/python-cwlgen.git
```

Comment:
Longer term can we get any of your changes into the upstream version?

Reply:
I opened a pull request and an issue to ask them to create a new release so that all of this can be directly pip installed from PyPi